ARCHITECTURE.MD

```diff
@@ -12,6 +12,8 @@ For the production release, [LXD](https://github.com/canonical/lxd) is pulling t
 
 In a clustered LXD environment the LXD-UI is available on all nodes, so you can query it from any cluster member. Authentication is shared between all nodes.
 
+LXD maintains different release branches in parallel. This includes long term support (LTS) releases which can have older versions of LXD bundled with LXD-UI. This means that LXD-UI should be compatible with older versions of LXD (v5.0 or later). Compatibility here means graceful degradation in the UI when certian features are missing in older LXD versions. 
+
```

Comment:
I was hoping we could describe how the test workflow is architected. Especially the matrix with channels and browsers. Also, how they are configured in `pr.yaml` and the connection to the projects in `playwright.config.ts`. Furthermore how the lxdVersion appears in the end to end tests and how it is used to enable / disable specific tests.

So instead of explaining why multiple version are there, explain how the different moving parts are working, and where they connect. Especially how they are configured, so we have a reference for future changes to the config.

Reply:
Aaaah sorry I misunderstood the assignment, will get that done :+1: